ocaml/testsuite/tests/language-extensions/language_extensions.ml

```diff
@@ -22,7 +22,7 @@ let report ~name ~text =
 
 let typecheck_with_extension ?(full_name = false) name =
   let success =
-    match Typecore.type_expression Env.initial_safe_string
+    match Typecore.type_expression (Lazy.force Env.initial_safe_string)
```

Comment:
This is kinda bogus. Because this whole test twiddles extensions, and yet the initial env won't change as the extensions are twiddled. It all means that the env of this `typecheck_with_extension` will be frozen in time at its first call, which is pretty awkward.

Reply:
Though I think it's probably OK, because no one will ever notice the problem. Still, there's something broken about this approach.

Reply:
As documented in the main thread, Chris and I agree both that this is kinda bogus and that it's OK to ship. But we should come back and fix later.